roles/intrution/templates/psad.conf

```diff
@@ -0,0 +1,28 @@
+# Configuration file for Psad Intrusion Detection System.
```

Comment:
panw panw tha mporouses na valeis kai to ansible_managed variable...Einai kalo practice gia templates gia na ksereis oti afto to arxeio einai generated apo to ansible